src/Pmi/Controller/SymfonyMigrationController.php

```diff
@@ -114,4 +115,13 @@ public function orderAction($participantId, $orderId, Application $app)
             $orderId
         ));
     }
+
+
+    /**
+     * @deprecated 2021-02-08
+     */
+    public function workqueue_indexAction(Application $app)
+    {
+        return $app->redirect('/s/workqueue');
```

Comment:
Missing the trailing slash. Symfony fixes this for you with the 301 redirect, but this breaks behind the WAF because Symfony uses an absolute URL for the redirect.

```suggestion
        return $app->redirect('/s/workqueue/');
```